mongo/integration/gridfs_test.go

```diff
@@ -45,6 +46,70 @@ func TestGridFS(x *testing.T) {
 		findIndex(findCtx, mt, mt.DB.Collection("fs.files"), false, "key", "filename")
 		findIndex(findCtx, mt, mt.DB.Collection("fs.chunks"), true, "key", "files_id")
 	})
+	// should not create a new index if index is numerically the same
+	mt.Run("numericallyEqualIndexes", func(mt *mtest.T) {
```

Comment:
nit: use a more descriptive test name like "indexes not created if equivalent indexes exist". I'm also trying to move away from the single-word camel-cased test names because English ones are much easier to read and `go test` combines them with underscores anyway. The exception here is function names like `UploadFromStream`, which are fine to use.